app/Http/Controllers/Api/DepartmentController.php

```diff
@@ -15,6 +15,8 @@ class DepartmentController extends Controller
      */
     public function index()
     {
-        return JsonResource::collection(Department::all());
+        $strategicDepartmentId = config('app.strategic_response_department_id');
+        $departments = Department::where('id', '<>', $strategicDepartmentId)->get();
+        return JsonResource::collection($departments);
```

Comment:
Won't this make it impossible for new hr advisors to register for this department?

Reply:
I suppose admins can still set it before upgrading the user to the HrAdvisor role.